### ISSUE TITLE
Can we color the fa icon color in the group headings?

### ISSUE THREAD

**Issue by jill88** [bastienwirtz/homer#806]
**Is your feature request related to a problem? Please describe.**
The group fa iconsuse the same color as the heading color.  But I'd like to change it.

**Describe the solution you'd like**
Use the link or link-hover color like the card icon colors can.

**Describe alternatives you've considered**
I tried the link and link-hover colors but it only works on the cards.

**Additional context**
Maybe an option to use one of the listed theme colors other than just the one option.

**Comment by l0f3n** [comment]
Yes! I would like to do this as well for the catppuccin theme, issue https://github.com/mrpbennett/catppuccin-homer/issues/9

**Comment by grasshide** [comment]
I did not like this either. Since it's bugging me for a while I did the following:

Add this to your config:
```
stylesheet:
   - "assets/custom.css"
```

Then create the file in the folder you set above. Put the following contents in the file:
```
@charset "UTF-8";

/* Custom icon color for the dark theme */
body #app.dark .group-title {
    --highlight-primary: #eaeaea;
}

/* Custom icon color for the light theme */
body #app .group-title {
    --highlight-primary: #eaeaea;
}
```
Of course you can change the color to whatever you like.
